daisy/validate.go

```diff
@@ -113,7 +108,9 @@ func (w *Workflow) validateVarsSubbed() dErr {
 		switch v.Interface().(type) {
 		case string:
 			if match := unsubbedVarRgx.FindStringSubmatch(v.String()); match != nil {
-				return errf("Unresolved var %q found in %q", match[0], v.String())
+				if !sourceVarRgx.MatchString(v.String()) {
```

Comment:
If we are iterating over unsubbedVars here, why do we even need validateVarsSubbed?